src/webhooks/__tests__/registry.test.ts

```diff
@@ -195,58 +195,58 @@ describe('ShopifyWebhooks.Registry.register', () => {
       deliveryMethod: 'Something else',
     };
 
-    const result = await ShopifyWebhooks.Registry.register(
+    const result = await shopify.webhooks.registry.register(
       webhook as RegisterOptions,
     );
     expect(result.PRODUCTS_CREATE.success).toBe(false);
   });
 
   it('only contains a single entry for a topic after an update', async () => {
-    fetchMock.mockResponseOnce(JSON.stringify(webhookCheckEmptyResponse));
-    fetchMock.mockResponseOnce(JSON.stringify(successResponse));
+    queueMockResponse(JSON.stringify(webhookCheckEmptyResponse));
+    queueMockResponse(JSON.stringify(successResponse));
     let webhook: RegisterOptions = {
       path: '/webhooks',
       topic: 'PRODUCTS_CREATE',
       accessToken: 'some token',
       shop: 'shop1.myshopify.io',
     };
-    await ShopifyWebhooks.Registry.addHandler('PRODUCTS_CREATE', {
+    await shopify.webhooks.registry.addHandler('PRODUCTS_CREATE', {
       path: '/webhooks',
       webhookHandler: genericWebhookHandler,
     });
-    await ShopifyWebhooks.Registry.register(webhook);
-    expect('PRODUCTS_CREATE' in ShopifyWebhooks.Registry.webhookRegistry);
+    await shopify.webhooks.registry.register(webhook);
+    expect('PRODUCTS_CREATE' in shopify.webhooks.registry.webhookRegistry);
 
     // Add a second handler
-    fetchMock.mockResponseOnce(JSON.stringify(webhookCheckEmptyResponse));
-    fetchMock.mockResponseOnce(JSON.stringify(successResponse));
+    queueMockResponse(JSON.stringify(webhookCheckEmptyResponse));
+    queueMockResponse(JSON.stringify(successResponse));
     webhook = {
       path: '/webhooks',
       topic: 'PRODUCTS_UPDATE',
       accessToken: 'some token',
       shop: 'shop1.myshopify.io',
     };
-    await ShopifyWebhooks.Registry.addHandler('PRODUCTS_UPDATE', {
+    await shopify.webhooks.registry.addHandler('PRODUCTS_UPDATE', {
       path: '/webhooks',
       webhookHandler: genericWebhookHandler,
     });
-    await ShopifyWebhooks.Registry.register(webhook);
-    expect('PRODUCTS_UPDATE' in ShopifyWebhooks.Registry.webhookRegistry);
-    expect(Object.keys(ShopifyWebhooks.Registry.webhookRegistry)).toHaveLength(
+    await shopify.webhooks.registry.register(webhook);
+    expect('PRODUCTS_UPDATE' in shopify.webhooks.registry.webhookRegistry);
+    expect(Object.keys(shopify.webhooks.registry.webhookRegistry)).toHaveLength(
       2,
     );
 
     // Update the second handler and make sure we still have the two of them
-    fetchMock.mockResponseOnce(JSON.stringify(webhookCheckResponse));
-    fetchMock.mockResponseOnce(JSON.stringify(successUpdateResponse));
+    queueMockResponse(JSON.stringify(webhookCheckResponse));
+    queueMockResponse(JSON.stringify(successUpdateResponse));
     webhook.path = '/webhooks/new';
-    await ShopifyWebhooks.Registry.register(webhook);
-    expect(Object.keys(ShopifyWebhooks.Registry.webhookRegistry)).toHaveLength(
+    await shopify.webhooks.registry.register(webhook);
```

Comment:
Personally, I feel like the interface would be a bit cleaner if we exported the registry _as_ `webhooks`, so we can call e.g. `shopify.webhooks.register()`.